### PR TITLE
prevent duplicates init

### PR DESCRIPTION
**Pull request by adalgrim**
Init

### CODE REVIEW

File: tokenizer.js [hunk 1]
```diff
@@ -212,6 +217,15 @@
...
 				return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
 			};
+			isFirstOccurrence = function(data) {
+			    var tokens = get();
+			    for (var i = 0; i < tokens.length; i++) {
+			        if (tokens[i] == data) {
+			            return false;
+			        }
+			    }
+				return true;
```
Comment (donmccurdy) [inline tokenizer.js]: this method can be shortened a bit:

```javascript
return get().indexOf(data) >= 0;
```

File: tokenizer.js [hunk 2]
```diff
@@ -29,7 +29,9 @@
 			source: null,
 			allowUnknownTags: true,
 			numToSuggest: 5,
-			onclick: null
+			onclick: null,
```
Comment (donmccurdy) [inline tokenizer.js]: I think this option is unused, perhaps left from another change?

File: tokenizer.js [hunk 3]
```diff
@@ -29,7 +29,9 @@
...
+			allowDuplicates: false,
+			suggestDuplicates: false
```
Comment (donmccurdy) [inline tokenizer.js]: let's just use the `allowDuplicates` option, and automatically not suggest duplicates if they're not allowed.

**Comment by donmccurdy** [comment]
This looks good thanks! Couple issues and then I think it's in good shape.

File: tokenizer.js (hunk 1 above)
Comment (adalgrim) [inline tokenizer.js]: You're right. I'll fix it.

File: tokenizer.js (hunk 3 above)
Comment (adalgrim) [inline tokenizer.js]: ok

File: tokenizer.js (hunk 2 above)
Comment (adalgrim) [inline tokenizer.js]: Could be :)

Comment (donmccurdy) [inline tokenizer.js]: hah oh, misread. carry on then!

**Comment by donmccurdy** [comment]
Looks good thanks!

**Pull request merged**